test/providers/hoverProvider.test.ts

```diff
@@ -131,7 +131,7 @@ function testModuleNames(
     {
       word: "ansible.builtin.debug -> msg",
       position: { line: 5, character: 10 } as Position,
-      doc: "The customized message that is printed. If omitted, prints a generic message.",
+      doc: "The customized message that is printed. If omitted\\, prints a generic message.",
```

Comment:
What is really this escaped backslash? It looks bit weird to me. Do we have a bug in documentation where a `\` is present before the comma?

Reply:
In MarkDown you can escape all punctuation, and to err on the safe side of caution the code is escaping all punctuation (https://github.com/ansible-community/antsibull-docs-parser/blob/main/src/antsibull_docs_parser/md.py#L19-L23). Likely not all of the characters that are now escaped need to be escaped, so this one might not be needed.

Reply:
I changed the escaping to also escape `.`, since that can actually make MarkDown change its behavior. I guess `,` doesn't really have to be quoted, at least as far as I understand the MarkDown specs it doesn't seem to have any impact. But I'm no MarkDown expert, so ...